stdlib/public/runtime/RuntimeAttribute.cpp

```diff
@@ -0,0 +1,40 @@
+//===---- AccessibleFunction.cpp - Swift protocol conformance checking ----===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// Checking and caching of Swift runtime attributes.
+//
+//===----------------------------------------------------------------------===//
+
+#include "ImageInspection.h"
+#include "Lazy.h"
+
+using namespace swift;
+
+// TODO: implement me!
+static Lazy<int> Functions;
```

Comment:
This file can be fleshed out like AcessibleFunction.cpp is. @Azoy says we can probably stub it out on macOS—the two `addImage` functions must _exist_ but can be no-ops.